hyper/ssl_compat_appengine.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+"""
+hyper/ssl_compat_appengine
+~~~~~~~~~
+
+Provides the ssl module interface which hyper assumes, based on AppEngine ssl.
+
+This module complements some constants and classes which don't exist on
+AppEngine's SSL module, to be used by hyper smoothly.
+See: https://cloud.google.com/appengine/docs/python/sockets/ssl_support
+"""
+
+import ssl
+
+from ssl import PROTOCOL_TLSv1, PROTOCOL_SSLv23, PROTOCOL_SSLv3
+from ssl import CERT_NONE, CERT_OPTIONAL, CERT_REQUIRED
+from ssl import match_hostname
```

Comment:
well, I'm afraid not, but it's available actually as far as I tried, and I think there's not a big reason to drop it now.
I can put try: with catching ImportError, but in that case I'm not sure how the fake match_hostname should be implemented.